app/controllers/admin/search/purchases_controller.rb

```diff
@@ -7,10 +7,23 @@ class Admin::Search::PurchasesController < Admin::BaseController
   def index
     @title = "Purchase results"
 
+    search_params = params.permit(:transaction_date, :last_4, :card_type, :price, :expiry_date, :purchase_status).to_hash.symbolize_keys
+
+    if search_params[:transaction_date].present?
+      begin
+        search_params[:transaction_date] = Date.strptime(search_params[:transaction_date], "%m/%d/%Y").to_s
+      rescue ArgumentError
+        flash[:alert] = "Please enter the date using the MM/DD/YYYY format."
+        @purchases = []
+        @service_charges = []
+        return
```

Comment:
I don't think will work now that we don't want to render the default Rails template?